jupyterhub/notebook-images/overlays/additional/minimal-gpu-notebook-imagestream.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   tags:
   - annotations:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.2.4"},{"name":"Notebook","version":"6.4"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.2"},{"name":"Notebook","version":"6.4"}]'
         openshift.io/imported-from: quay.io/modh/odh-minimal-notebook-container
```

Comment:
Could you change also the field openshift.io/imported-from to ->  `openshift.io/imported-from: quay.io/modh/cuda-notebooks` ?

Reply:
Overall, the change in the JupyterLab version looks good to me

Reply:
done

Reply:
@lucferbux could you review the PR as well? I don't have permission to merge it